setup.sh

```diff
@@ -100,9 +100,9 @@ function check_dependencies()
     # Installation failed...
     if [[ "$ret" -ne 0 ]]; then
       error_message='[ERROR]: Dependency installation failure:'
-      [[ "$ret" -eq 1 ]] && error_message+='\nLacked sudo privileges to install the packages'
-      [[ "$ret" -ne 1 ]] && error_message+='\nSomething went wrong when installing the packages'
-      error_message+='\n'"$package_list"
+      [[ "$ret" -eq 1 ]] && error_message+=' Lacked sudo privileges to install the packages:'
+      [[ "$ret" -ne 1 ]] && error_message+=' Something went wrong when installing the packages:'
+      error_message+="\n$package_list"
```

Comment:
The assignment `error_message+="\n$package_list"` doesn't add a newline before the value of `package_list`. `complain "$error_message"` would result in something like

```bash
[ERROR]: Dependency installation failure: Lacked sudo privileges to install the packages:\n<list of packages>
```

Instead, you should use `error_message+=$'\n'"$package_list"` to add this leading newline.

To better understand the why of it see this excerpt from the Bash reference manual below:

<img width="785" alt="Screen Shot 2023-10-30 at 22 49 58" src="https://github.com/kworkflow/kworkflow/assets/95055784/fa5f8380-abac-44bf-9e5b-c637eba23abe">